core/src/main/java/com/yahoo/oak/BlocksPool.java

```diff
@@ -97,6 +97,18 @@ static void setBlockSize(int blockSize) {
         }
     }
 
+    static void clear() {
```

Comment:
Is it some new API we expose outside of Oak? If yes, we should update README. If not, then we probably should expose it somehow, because it might be useful for some cases. However, it should be well explained.

Reply:
Currently, `BlocksPool` is not exposed outside. So this change is not seen by the user.
Exposing `BlocksPool` functionality to the user should be discussed, but not as part of this PR.